### DISCUSSION TITLE
Will the scope of aiocogeo-rs include plain old GeoTIFFs (not COGs)?

### DISCUSSION THREAD

**Discussion by maxrjones** [developmentseed/async-tiff#3] [Q&A] [answered]
I'm looking at options for extracting metadata and internal chunk byte ranges from GeoTIFFs for VirtualiZarr and am wondering if a Python bindings for aiocogeo-rs could be a long-term option?

**Comment by kylebarron** [comment, marked as answer]
For some reason my notifications weren't set to "all" here, so I never saw this. 

Yes, the scope of this library should be COG and GeoTIFF-first, but in principle should be able to support most types of TIFF images. Though some obscure versions of tiff files may not be supported, at least at first.
